draft-ietf-quic-transport.md

```diff
@@ -1847,6 +1847,11 @@ endpoint migrates to a new address.
 An endpoint MUST NOT initiate connection migration before the handshake is
 finished and the endpoint has 1-RTT keys.  The design of QUIC relies on
 endpoints retaining a stable address for the duration of the handshake.
+Clients MUST NOT initiate connection migration before they are
```

Comment:
Just a nit, but this appears to put stricter requirements on the statements made in the previous two sentences, is there any reason not to merge them together?

Reply:
I think the updated version solves your issue.